src/peer/negotiator.js

```diff
@@ -550,8 +550,13 @@ class Negotiator extends EventEmitter {
    */
   _replacePerStream(newStream) {
     const localStreams = this._pc.getLocalStreams();
+
+    // temporarily unset onnegotiationneeded so that it doesn't do anything on removeStream.
```

Comment:
May I add one more comment here? Like, 

```js
// temporarily unset onnegotiationneeded so that it doesn't do anything on removeStream,
// which causes extra offer/answer negotiation and makes the signaling state unexpected.
```